src/components/navbar.tsx

```diff
@@ -1,5 +1,96 @@
+import Image from "next/image";
+import Link from "next/link";
+
 export default function Navbar() {
+  const categoriesNav = [
+    "novidades",
+    "sapatos",
+    "botas",
+    "acessórios",
+    "promoções",
+    "brizza",
+  ];
+
+  const categoryBrizza = categoriesNav.pop();
+  const categoriesWithoutBrizza = categoriesNav.filter(
+    (category) => category !== "brizza",
```

Comment:
Sempre que temos uma string ou número que vai ser hard coded, usamos ela em uma const em uppercase tbm. Dessa forma, se usarmos em mais lugares nesse componente, modificamos em um unico lugar.
Se for um número, por exemplo, com uma const damos semântica ao número, por exemplo:

````jsx
  const TIME_TO_RELOAD_IN_MS = 100

  setTimeout(() => {
    //do something
  }, TIME_TO_RELOAD_IN_MS)